test/state/bloom.cpp

```diff
@@ -0,0 +1,58 @@
+// evmone: Fast Ethereum Virtual Machine implementation
+// Copyright 2023 The evmone Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "bloom.hpp"
+#include "state.hpp"
+#include <intx/intx.hpp>
+
+namespace evmone::state
+{
+
+namespace
+{
+/// Adds entry to bloom filter data
+/// based on
+/// https://ethereum.github.io/execution-specs/autoapi/ethereum/shanghai/bloom/index.html?highlight=add_to_bloom#add-to-bloom
```

Comment:
maybe URL without highlight is more concise 
```suggestion
/// https://ethereum.github.io/execution-specs/autoapi/ethereum/shanghai/bloom/index.html#add-to-bloom
```